rero_ils/dojson/utils.py

```diff
@@ -222,6 +339,16 @@ def get_field_items(value):
         return utils.iteritems(value)
 
 
+def build_string_from_subfields(value, subfield_selection, separator):
```

Comment:
interesting to use a default value for `separator` ?

Reply:
Bonne idée!
Je vais ajouter un `separator`par default.